### ISSUE TITLE
太棒的思想了，看源码有个问题

### ISSUE THREAD

**Issue by dayu1979** [GGGHSL/InfoSwap-master#2]
iib.py里面 191行，标准差为什么取的是均值呢？不太懂
            m_s = torch.mean(Rs, dim=0)  # [C, H, W]
            std_s = torch.mean(Rs, dim=0)
            Rs_params.append([m_s, std_s])
            eps_s = torch.randn(size=Rt.shape).to(Rt.device) * std_s + m_s
            feat_t = Rt * (1. - lambda_t) + lambda_t * eps_s
            Xt_feats.append(feat_t)  # only related with lambda

            m_t = torch.mean(Rt, dim=0)  # [C, H, W]
            std_t = torch.mean(Rt, dim=0)
            Rt_params.append([m_t, std_t])
            eps_t = torch.randn(size=Rs.shape).to(Rs.device) * std_t + m_t
            feat_s = Rs * (1. - lambda_s) + lambda_s * eps_t
            Xs_feats.append(feat_s)  # only related with lambda

**Comment by GGGHSL** [comment]
Thank you for reminding me of this.

**Comment by lyx0208** [comment]
这个问题解决了么 @dayu1979 @GGGHSL

**Comment by yanghuiying1002** [comment]
我也有一样的问题，但如果尝试把mean改成std生成不出图片

**Comment by yanghuiying1002** [comment]
请问这个问题有解答吗
